### PR TITLE
make Promrec container use log rotation and run using environment variables

### PR DESCRIPTION
**Pull request by bitzge**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,4 +17,14 @@ WORKDIR /promqueen
 
 COPY --from=build $GOPATH/bin/promrec /promqueen
 
-ENTRYPOINT ["/promqueen/promrec"]
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y logrotate findutils && rm -rf /var/lib/apt/lists/*
```
Comment (theangi) [inline Dockerfile:21]: Better using `apt-get` for both update and install

File: README.md [hunk 2]
```diff
@@ -76,6 +76,17 @@ Flags:
...
+### Environment variables
+
+```
+ROTATION_* variables are used to configure log rotation with logrotate
+
+      ROTATION_FILE_LOG: "/var/log/cleafy/metrics/metrics.prom" # must be the some of output parameter in PROM_ARGS envVar
+      ROTATION_PERIOD: daily
+      ROTATION_COUNT: 10 # after 10 periods will delete older log files
+      ROTATION_SIZE: -1
+      PROM_ARGS: "--output=/var/log/cleafy/metrics/metrics.prom --interval=30s -u serviceName1=URL1 -u serviceName2=URL2"
```
Comment (theangi) [inline README.md:88]: if the above are indented for the rotation, this is not related to rotation. I would propose something like this:


- `PROM_ARGS`: The argument for the promqueen service. Output, interval and at least one service is mandatory. E.g. `--output=/var/log/cleafy/metrics/metrics.prom --interval=30s -u serviceName1=URL1 -u serviceName2=URL2 ...`
- `ROTATION_FILE_LOG`:  where the rotation should occurr. Must be the same of the "output" parameter in PROM_ARGS
- `ROTATION_PERIOD`: how frequently a rotation will occurr. E.g. "daily"
- `ROTATION_COUNT`: how many rotation will be retained. E.g. 10
- `ROTATION_SIZE`: how big each rotation file will be in bytes. -1 means no limit
```

Maybe we should provide also an example `docker run` command with the least amount of variables needed to start it, what do you think?

**Review by theangi — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (bitzge) [reply README.md]: thank you for feedback, looks better and clearer!

**Review by theangi — APPROVED** [review]
Not a fan of the job every five minute, but seems to work. approved!

**Pull request merged**